curation/main.py

```diff
@@ -0,0 +1,115 @@
+from fastapi import FastAPI, Query, Request
+from fastapi.staticfiles import StaticFiles
+from typing import Dict
+from curation import version
+from curation.schemas import NormalizeGeneResponse, DomainIDResponse, ExonCoordsRequest, \
+    ExonCoordsResponse, SequenceIDResponse, FusionValidationResponse
+from curation.gene_services import get_gene_id
+from curation.domain_services import get_domain_id
+from curation.uta_services import postgres_instance, get_genomic_coords
+from curation.sequence_services import get_ga4gh_sequence_id
+from curation.validation_services import validate_fusion
+
+
+app = FastAPI(version=str(version))
+
+@app.on_event('startup')
+async def startup():
+    await postgres_instance.create_pool()
+    app.state.db = postgres_instance
+
+
+@app.get('/lookup/gene',
+         operation_id='normalizeGene',
+         response_model=NormalizeGeneResponse)
```

Comment:
I think adding `response_model_exclude_none=True` to each route would be nice so we don't have to see a lot of nulls
```suggestion
         response_model=NormalizeGeneResponse,
         response_model_exclude_none=True)
```